sensor_msgs/RegionOfInterest bounding_box
float32 range
